ignite/pkg/cosmosutil/gentx.go

```diff
@@ -7,9 +7,10 @@ import (
 
 	sdkmath "cosmossdk.io/math"
 
-	"github.com/cometbft/cometbft/crypto/ed25519"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
```

Comment:
```suggestion
```

Reply:
no, that's the gci linter :D

Reply:
https://github.com/ignite/cli/blob/main/.golangci.yml#L47-L53